client/my-sites/patterns/components/pattern-library/style.scss

```
@@ -12,6 +12,10 @@
 	top: 0;
 	z-index: 3;
```

```suggestion
	z-index: 3;
	transition: box-shadow 0.2s linear;
```
Nit, but a small transition would be nice.